packages/plugin-data/package.json

```diff
@@ -65,21 +65,22 @@
     "test": "sf-test"
   },
   "dependencies": {
-    "@oclif/config": "^1",
     "@salesforce/command": "^3.0.3",
     "@salesforce/core": "^2.13.0",
-    "@salesforce/data": "^0.1.1",
     "@salesforce/ts-types": "^1.4.3",
+    "chalk": "^4.1.0",
```

Comment:
What is chalk being used for? I think it can be accessed via `this.ux` in the command

Reply:
@WillieRuemmele, chalk is being used to apply display attributes to text that is written to the console. Display attributes (color, bolding, etc) in toolbelt was accomplished by accessing the color property in the available logger instance. That property does not seem to be present on the available Logger class from @salesforce/core.

Open to suggestions and edification :).

Reply:
chalk was removed from the `UX` class a while ago on purpose.  Not everyone needs styling with their terminal output and it's easy to add the dependency to use it.  This is the expected approach for adding styling.